airbyte-integrations/connectors/source-fastbill/source_fastbill/components.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+import base64 
+from airbyte_cdk.sources.declarative.auth.token import BasicHttpAuthenticator
+from airbyte_cdk.sources.declarative.types import Config
+from isodate import Duration, parse_duration
+
+
+@dataclass
+class CustomAuthenticator(BasicHttpAuthenticator):
+
+    config: Config
+    @property
+    def token(self) -> str:
```

Comment:
Please add a unit test to validate the custom function.

Reply:
@marcosmarxm added unit tests